Description: Don't throw errors when starting the backend 
 if no tuner was defined.  The user can configure it later.
 .
 This is intended to allow mythtv-backend to run with a fresh install.
 .
 Upstream doesn't agree with this direction, but it's a priority for us.
Author: Mario Limonciello <superm1@ubuntu.com>
Bug: http://code.mythtv.org/trac/ticket/9626

Index: mythtv/mythtv/programs/mythbackend/scheduler.cpp
===================================================================
--- mythtv.orig/mythtv/programs/mythbackend/scheduler.cpp
+++ mythtv/mythtv/programs/mythbackend/scheduler.cpp
@@ -179,8 +179,7 @@ bool Scheduler::VerifyCards(void)
         LOG(VB_GENERAL, LOG_ERR, LOC +
                 "No capture cards are defined in the database.\n\t\t\t"
                 "Perhaps you should re-read the installation instructions?");
-        error = GENERIC_EXIT_SETUP_ERROR;
-        return false;
+        return true;
     }
 
     query.prepare("SELECT sourceid,name FROM videosource ORDER BY sourceid;");
